docs/operator-manual/declarative-setup.md

```diff
@@ -655,7 +655,7 @@ stringData:
 
 ## Resource Exclusion/Inclusion
 
-Resources can be excluded from discovery and sync so that Argo CD is unaware of them. For example, `events.k8s.io` and `metrics.k8s.io` are always excluded. Use cases:
+Resources can be excluded from discovery and sync so that Argo CD is unaware of them. For example, the apiGroup/kind `events.k8s.io/*`, `metrics.k8s.io/*`, `coordination.k8s.io/Lease`, and `""/Endpoints` are always excluded. Use cases:
```

Comment:
I don't see `""/Endpoints` in the list of core exclusions, should it be `""/Event` instead?

https://github.com/argoproj/argo-cd/blob/8840929187f4dd7b9d9fd908ea5085a006895507/util/settings/resources_filter.go#L7

```suggestion
Resources can be excluded from discovery and sync so that Argo CD is unaware of them. For example, the apiGroup/kind `events.k8s.io/*`, `metrics.k8s.io/*`, `coordination.k8s.io/Lease`, and `""/Event` are always excluded. Use cases:
```

Reply:
@morey-tech 
Thank you for reviewing the PR.
Here is the code reference
- https://github.com/argoproj/gitops-engine/blob/c0ffe8428a23011ae9f6d0d3d01f9bf6200a4339/pkg/cache/cluster.go#L1076-L1078
- https://github.com/argoproj/gitops-engine/blob/c0ffe8428a23011ae9f6d0d3d01f9bf6200a4339/pkg/cache/cluster.go#L1050-L1054

Reply:
It seems like `Endpoints` resource isn't excluded using the resource filter mechanism but rather a separate one in the gitops engine. Not sure it makes sense to include it in the example of what is always excluded using the resource filter.

Reply:
The `Endpoints` resource is filtered when the resource update event comes in. Ref to [code](https://github.com/argoproj/argo-cd/blob/8840929187f4dd7b9d9fd908ea5085a006895507/controller/state.go#L445).

The `events.k8s.io/*`, `metrics.k8s.io/*`, `coordination.k8s.io/Lease` resources are filtered when comparing the live state versus git manifest. Ref to [code](https://github.com/argoproj/argo-cd/blob/8840929187f4dd7b9d9fd908ea5085a006895507/controller/state.go#L457).

Reply:
Since this section of the docs is for the Resource Exclusion feature and not generally about filtered resources, I'm hesitant to include `Endpoints` in the list, even though they are also excluded (but in a different context). Ultimately, it's okay since the user will know they are excluded and can't change that.

Reply:
Because the purpose of this reference is just to give an example, it doesn't need to be a compressive list. But I would leave it as is because these resources are excluded by default.

Reply:
Because we are working on tuning the app controller CPU high issue.
It'd be good to doc existing excluded resources rather than trace the source code.